### ISSUE TITLE
Opening files that do not exist panics

### ISSUE THREAD

**Issue by shingtaklam1324** [shingtaklam1324/logic-rs#5]
### Actions preceding
Opening a file that does not exist
## Actual Behaviour
Program panics with following error code
```
thread 'main' panicked at 'Error opening file: Error { repr: Os { code: 2, message: "No such file or directory" } }', /checkout/src/libcore/result.rs:906:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```
## Expected Behaviour
Graceful Exit with error printed out
### Version
0.2.2
### Target Triple
Global

**Comment by shingtaklam1324** [comment]
An option would be to change the return of `open_file::open_file` to `Result<String, &'static str>`

**Comment by shingtaklam1324** [comment]
Fixed by pull request #6
